.circleci/config.yml

```
@@ -227,7 +227,7 @@ jobs:
   check_code_1_16:
     working_directory: /go/src/github.com/stellar/go
     docker:
-      - image: golang:1.16
+      - image: golang:1.16.5
```

Maybe we should replace it with:
```suggestion
      - image: golang:1.16
```
so that it will always pull latest Golang 1.16.x. This was recommended by @jacekn and @satyamz in the issue about bumping Golang versions in SDF. Same thing for 1.15.x.

---

I like this idea. We basically always want to be on latest patch release. Let's do it.

For minor versions let's stick to manual update for now.

---

Cool, I didn't realize 1.16 would always be the latest

---

Done!